src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -31,6 +31,25 @@
     <Prefer32Bit>false</Prefer32Bit>
     <UseVSHostingProcess>true</UseVSHostingProcess>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
I think the existing configuration needs to target x64. No need to add new configurations.